app/Cron/Nightly/SetActiveFlights.php

```diff
@@ -43,10 +43,7 @@ public function checkFlights(): void
             if (!$flight->active) {
                 continue;
             }
-
-            // Set to visible by default
-            $flight->visible = true;
```

Comment:
Are you sure this should be removed ? 
Since you have changed the query above, we are necessarily in the case where the flight belongs to v7, so why change the logic?

Reply:
This should be the right logic